src/main/sensors/gyro.c

```diff
@@ -629,7 +629,7 @@ static bool isOnFinalGyroCalibrationCycle(const gyroCalibration_t *gyroCalibrati
 
 static uint16_t gyroCalculateCalibratingCycles(void)
 {
-    return (CALIBRATING_GYRO_CYCLES / gyro.targetLooptime) * CALIBRATING_GYRO_CYCLES;
```

Comment:
Measurement units of the original code are questionable. What was it, `[cycles / usec]`?
Besides, it was obviously overflowing the `uint16_t` at any rates present in BF, e.g. for 8KHz it was:
`10k / 125 * 10k = 800k`

Reply:
It was working for 0.5 KHz though :-)